core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -13,6 +13,7 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import javax.annotation.concurrent.ThreadSafe;
```

Comment:
Add an empty line between line 15 and 16?
`javax.*` imports should be separated with `java.*` imports by an empty line.
http://www.alluxio.org/docs/master/en/Contributing-to-Alluxio.html#coding-style